app/helpers/social_helper.rb

```diff
@@ -13,4 +13,9 @@ def answer_share_url(answer)
       protocol: (APP_CONFIG["https"] ? :https : :http),
     )
   end
+
+  def answer_copy_content(answer)
+    # copy content template: "question - answer [link]"
+    "#{answer.question.content} - #{answer.content} [#{answer_share_url(answer)}]"
+  end
```

Comment:
RDoc comments go above the method instead of inside it.

```suggestion
  # copy content template: "question - answer link"
  def answer_copy_content(answer)
    "#{answer.question.content} - #{answer.content} #{answer_share_url(answer)}"
  end
```

This can also be shortened using end-less method definition like so:
```suggestion
  # copy content template: "question - answer link"
  def answer_copy_content(answer) = "#{answer.question.content} - #{answer.content} #{answer_share_url(answer)}"
```

Reply:
- Thanks for your suggestion. I'd keep this in mind for my future PRs.
- As per [this comment](https://github.com/Retrospring/retrospring/pull/1461#discussion_r1390304946), I'll be removing this method. Thanks. :)